data/json/overmap/overmap_special/specials.json

```diff
@@ -601,7 +605,7 @@
     "city_distance": [ 5, -1 ],
     "city_sizes": [ 1, -1 ],
     "occurrences": [ 5, 100 ],
-    "flags": [ "UNIQUE" ]
+    "flags": [ "CLASSIC", "FARM" ]
```

Comment:
Please don't remove the `UNIQUE` flag. With this flag, the chance to spawn will be the intended 5 out of 100. Without this flag, this farm will spawn minimum 5 and maximum 100 times on a single overmap.

Reply:
That looks like a copy-paste error, it shouldn't be getting removed anywhere